### PR TITLE
docs: parachain onboarding locally

### PR DESCRIPTION
**Pull request by Daanvdplas**
The following guide explains how to launch a parachain to a local test network. The guide `launching-on-paseo.md` is almost and hasn't been edited. I think this should be removed.

There is still a lot of room for improvement but not the scope of this work.

### CODE REVIEW

**Comment by Daanvdplas** [comment]
[sc-1711]

**Comment by Daanvdplas** [comment]
[sc-1778]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:54]: Again, why remove it?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:67]: Use `pop call chain`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:60]: Use `pop call chain`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:67]: Why the call_data here? Is not clear what we are doing if we specify the extrinsic we are calling?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: Why remove this part?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: Is not interesting to keep this?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:433]: Output is outdated.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:425]: That's wrong no?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:63]:
```suggestion
> Note: we are calling the specified rpc port `57731` which is specified in the created `network.toml` file to interact the validator `alice`
```

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: You feel it is necessary?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:54]: You feel it is necessary? Someone wants to deploy a parachain, there is already a lot of info.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: Do we need it? We do everything with pop cli

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: See the bash script saying exactly the same

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:427]: Why?

Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:427]: Here is the Secret key URI, not the public. maybe just leave it as `<STASH ACCOUNT>`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:232]: You are right, sorry

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:166]: True!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (AlexD10S) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: I don't see it as a bad content a brief explanation on what Paseo is, maybe is too long. But not blocker for me.

Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:9]: Moved it back

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Updated

**Review by AlexD10S — APPROVED** [review]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:250]: I am a bit skeptical about inviting users to modify the plain spec directly instead of having code in the project that gives them the expected genesis.

But I think this is a different conversation than what this PR involves.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:440]: We could include `--use-wallet` to exemplify a more solid flow.
Otherwise we can come back once the feature its merged!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:476]: Same comment than above regarding `--use-wallet`.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:499]: Not 100% correct. One only buy the finalization of one core.
More of a pay finalization as you produce blocks.

Buying a core would be something more akin to buying bulk coretime. Which in reality isn't cores either what is being sold, but regions. :)

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:497]: Might be interesting pointing to the wiki - https://wiki.polkadot.network/docs/learn-agile-coretime
As a reference in case someone arrives to this point and is not aware of what on demand coretime is. :)

**Review by al3mart — CHANGES_REQUESTED** [review]

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]:
```suggestion
  * Use this guide to mimic the manual onboarding process for a Paseo Local Testnet.&#x20;
```

As we call a production Paseo network `Paseo Live Testnet`, I suggest following the format `Paseo [Live | Local] Testnet` to keep the consistency.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:21]: And then here it is called `Paseo Testnet`, a bit confusing to me whether this is Live Testnet or Local Testnet.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: I guess this is Paseo Live Testnet?

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:16]: Here it is used `Paseo Local Testnet`

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:29]:
```suggestion
## Launch Paseo Testnet on Local
```

or 

```suggestion
## Launch Paseo Local Testnet
```

**Review by chungquantin — CHANGES_REQUESTED** [review]
We should choose a consistent naming convention for "Paseo Testnet" throughout the document. For clarity and readability, I suggest using `Paseo [Live | Local] Testnet` as the standard term in all instances.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (chungquantin) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:92]:
```suggestion
> A [collator](https://wiki.polkadot.network/docs/learn-collator) is the parachain node that will be running for your parachain.
```

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:250]: Yes out of scope I'm afraid

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:499]: validated and finalised*

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:497]: Nice one!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: Yes, great suggestion

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:92]: Great suggestion, thank you!

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:19]: I am calling it Paseo Local and Paseo, the latter being live. Also removing the word testnet as that is explained via a link at the top.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (evilrobot-01) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:220]: Do we still need this output if we have the gif?

Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:214]: For copy pasting it can still be handy perhaps.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (evilrobot-01) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:491]: 'Buy a core' doesnt sound right to me - we are acquiring coretime for a single block where this sounds like you have bought it outright and own it, which is not true.

https://wiki.polkadot.network/docs/learn-agile-coretime#on-demand-coretime

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:371]: Note that the above node key is placed in the `.data/` directory assuming that the user will run the collator copy/pasting the instructions.

More concretely the node key generated in the previous step will only be used if `--base-path ./data ` is provided.

Authorities need to be run with a specific node key. Otherwise `--unsafe-force-node-key-generation` can be provided to delegate the key generation to the node itself (this presents other nuances). But this is OK for local environments.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/running-on-paseo-locally.md:10]: Afraid of adding content to this already long doc. 
But I feel we could use some extra pointers for the bullet points bellow.

#1. Is very useful when testing the runtime of your chain. You are not really worried about how the onboarding process works. But you need to see the runtime executing some blocks. "live" integration tests with other chains and such.

#2. Is only helpful when you are trying to learn how to onboard your chain to an already running relay chain (like Westend, Paseo, Kusama or Polkadot). It is very helpful to nail down the concepts around bootstrapping collators, seeing the process and taking a few leanings from it.
This document.

The learnings in #2. are mostly equivalent to #3. in terms of what the user needs to know.
I'd argue that we can port most of the new content from here into 3. (or vice versa) and get rid of all the details around configuring  the relay chain. It is too nuanced, and possibly it changes in future versions of the SDK.

Ideally no user would need to know about the scheduling gymnastics the relay does.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md:214]: You are right, removed the command and output.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: This was a bit hard to follow what you meant. Please see the latest changes and shout if there is anything pressing missing (or create an issue if not pressing).

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Added a little `Note`

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md
Comment (Daanvdplas) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:1]: Didn't change much in this guide, just copy pasted.

File: pop-cli-for-appchains/guides/spinning-up-a-polkadot-network/spinning-up-the-polkadot-network.md
Comment (Daanvdplas) [inline pop-cli-for-appchains/guides/spinning-up-a-polkadot-network/spinning-up-the-polkadot-network.md:7]: Added configuration to this file. As we would like to create a command `pop up paseo` this guide will become quickly irrelevant.

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:7]:
```suggestion
In order to get a chain's block validated and finalised by the Relay chain it needs to acquire [coretime](https://wiki.polkadot.network/docs/learn-agile-coretime).
```

File: pop-cli-for-appchains/guides/launch-a-chain/coretime.md
Comment (al3mart) [inline pop-cli-for-appchains/guides/launch-a-chain/coretime.md:8]: "pop allows users to obtain on demand coretime as easily as executing one command:"

**Review by al3mart — APPROVED** [review]
This is way more digestible! :+1: 

Left a couple of comments, but I'll leave it up to you to include, @Daanvdplas !
Good job.

File: pop-cli-for-appchains/guides/running-on-paseo-locally.md
Comment (Daanvdplas) [reply pop-cli-for-appchains/guides/running-on-paseo-locally.md]: Changed to acquire a core

**Review by al3mart — APPROVED** [review]

**Pull request merged**
